CHANGELOG.rst

```diff
@@ -30,9 +30,30 @@ Improved terminal support
 fish 3.2.1 (released March 18, 2021)
```

Comment:
you should rebase on master, instead of merging it to your branch - that way only your changes will show up in the PR

Reply:
Thanks 👍 I'm no git expert.